reduce_size.py

```diff
@@ -0,0 +1,117 @@
+import concurrent.futures
+import datetime
+import pandas as pd
+from glob import glob
+from tardis_dev import datasets, get_exchange_details
+import os
+import argparse
+
+COLUMNS = ["symbol", "timestamp", "type", "strike_price", "expiration", "underlying_price", "bid_price", "bid_amount",
+           "ask_price", "ask_amount", "mark_price", "mark_iv", "delta", "theta"]
+
+COLUM_TYPES = {
+    'symbol': str,
+    'timestamp': int,
+    'type': str,
+    'strike_price': float,
+    'expiration': int,
+    'underlying_price': float,
+    'bid_price': float,
+    'bid_amount': float,
+    'ask_price': float,
+    'ask_amount': float,
+    'mark_price': float,
+    'delta': float,
+    'theta': float
+}
+
+def get_files(folder='datasets'):
+    """Get the *.csv.gz files from the folder"""
+    files = glob(f"{folder}/*.csv.gz")
```

Comment:
you mentioned you might want to refactor the file type that files are imported in, what did you have in mind?

Reply:
On my machine using the reduced data it takes ~12 seconds to read one of the *.csv.gz files into a df. I haven't done the testing yet but I would expect using something like hdf5 or parquet would speed up the data loading.

Reply:
article I found comparing compression methods https://towardsdatascience.com/the-best-format-to-save-pandas-data-414dca023e0d